modules/host/persist.go

```diff
@@ -142,6 +177,11 @@ func (h *Host) establishDefaults() error {
 	}
 	h.spaceRemaining = h.settings.TotalStorage
 
+	// Reduce the window size when testing.
+	if build.Release == "testing" {
+		h.settings.WindowSize = testingWindowSize
+	}
```

Comment:
alternatively, define `testingWindowSize` conditionally based on `build.Release`, the way the renter does with its defaults. That way you don't need to worry about the discrepancy between `h.settings.WindowSize` and `testingWindowSize`.